boards/arm/frdm_kw41z/Kconfig.board

```diff
@@ -7,3 +7,6 @@ config BOARD_FRDM_KW41Z
 	bool "NXP FRDM-KW41Z"
 	depends on SOC_SERIES_KINETIS_KWX
 	select SOC_PART_NUMBER_MKW41Z512VHT4
+	# Since Voltage Reference block is not supported, the ADC does not work
+	# properly
```

Comment:
I based this comment on the commit message in commit 80aec78 ("boards: frdm_kw41z: set VREF to alternate source").